projects/rocprofiler-systems/docker/containers-ci.yml

```
@@ -0,0 +1,23 @@
+# Supported OS combinations for continuous integration
+
+matrix:
+  # ubuntu
+  - distro: "ubuntu"
+    version: "22.04"
+  - distro: "ubuntu"
+    version: "24.04"
+  # opensuse
+  - distro: "opensuse"
+    version: "15.5"
+  - distro: "opensuse"
+    version: "15.6"
+  # RHEL8
+  - distro: "rhel"
+    version: "8.10"
+  # RHEL9
+  - distro: "rhel"
+    version: "9.3"
+  - distro: "rhel"
+    version: "9.4"
+  - distro: "rhel"
+    version: "9.5"
```

```suggestion
    version: "9.5"
    
```
Formatting. Ending file with a new line.